src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -434,6 +485,73 @@ const LibraryAuthoringPageHeader = connect(
   },
 )(injectIntl(LibraryAuthoringPageHeaderBase));
 
+const ContentTagsDrawer = ({ openContentTagsDrawer, setOpenContentTagsDrawer }) => {
+  const iFrameRef = useRef();
+
+  if (openContentTagsDrawer) {
+    document.body.classList.add('drawer-open');
+  } else {
+    document.body.classList.remove('drawer-open');
+  }
+
+  useEffect(() => {
+    const handleCloseMessage = (event) => {
+      if (event.data === 'closeManageTagsDrawer') {
+        setOpenContentTagsDrawer('');
+      }
+    };
+
+    const handleCloseEsc = (event) => {
+      if (event.key === 'Escape' || event.keyCode === 27) {
+        setOpenContentTagsDrawer('');
+      }
+    };
+
+    // Add event listener to close drawer when close button is clicked or ESC pressed
+    // from within the Iframe
+    window.addEventListener('message', handleCloseMessage);
+    // Add event listern to close the drawer when ESC pressed and focus outside iframe
+    // If ESC is pressed while the Iframe is in focus, it will send the close message
+    // to the parent window and it will be handled with the above event listener
+    window.addEventListener('keyup', handleCloseEsc);
+
+    return () => {
+      window.removeEventListener('message', handleCloseMessage);
+      window.removeEventListener('keyup', handleCloseEsc);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (openContentTagsDrawer && iFrameRef.current) {
+      iFrameRef.current.focus();
+    }
+  }, [openContentTagsDrawer]);
+
+  // TODO: The use of an iframe in the implementation will likely change
+  const renderIFrame = () => (
+    <iframe
+      ref={iFrameRef}
+      title="manage-tags-drawer"
+      className="w-100 h-100 border-0"
+      src={`${getConfig().COURSE_AUTHORING_MFE_BASE_URL}/tagging/components/widget/${openContentTagsDrawer}`}
```

Comment:
Hi @yusuf-musleh!
Could we change this var to `COURSE_AUTHORING_MICROFRONTEND_URL`?

The `tutor-mfe` plugin already define it by default:
https://github.com/overhangio/tutor-mfe/pull/182

Reply:
@rpenido Sure thing, updated here: https://github.com/openedx/frontend-app-library-authoring/pull/400/commits/83f12d8149d6c658913800836da7b2d0bb545180